playbooks/monitoring/kibana/docs_compare.py

```diff
@@ -30,29 +30,27 @@ def handle_special_case_kibana_settings(legacy_doc, metricbeat_doc):
     legacy_doc["kibana_settings"]["xpack"].pop("default_admin_email")
 
 def handle_special_case_kibana_stats(legacy_doc, metricbeat_doc):
-  # Special case this until we have resolution on https://github.com/elastic/kibana/pull/70677#issuecomment-662531529
-  metricbeat_doc["kibana_stats"]["usage"]["search"]["total"] = legacy_doc["kibana_stats"]["usage"]["search"]["total"]
-  metricbeat_doc["kibana_stats"]["usage"]["search"]["averageDuration"] = legacy_doc["kibana_stats"]["usage"]["search"]["averageDuration"]
-
   # Special case for https://github.com/elastic/kibana/pull/76730
   # To be removed if/when https://github.com/elastic/beats/issues/21092 is resolved
   metricbeat_doc["kibana_stats"]["os"]["cpuacct"] = legacy_doc["kibana_stats"]["os"]["cpuacct"]
   metricbeat_doc["kibana_stats"]["os"]["cpu"] = legacy_doc["kibana_stats"]["os"]["cpu"]
 
-def filter_kibana_usage_stats(legacy_doc, metricbeat_doc):
-  for i in unused_kibana_usage_properties: 
-    legacy_doc["kibana_stats"]["usage"][i] = metricbeat_doc["kibana_stats"]["usage"][i]
+    
 
 def handle_special_cases(doc_type, legacy_doc, metricbeat_doc):
     if doc_type == "kibana_settings":
         handle_special_case_kibana_settings(legacy_doc, metricbeat_doc)
     if doc_type == "kibana_stats":
+        # Starting in 7.11, we want to stop storing `usage` data within Kibana monitoring documents
+        # https://github.com/elastic/kibana/issues/83521
+        legacy_doc["kibana_stats"]["usage"] = False
+        metricbeat_doc["kibana_stats"]["usage"] = False
```

Comment:
Maybe we should just delete it instead?
```suggestion
        del legacy_doc["kibana_stats"]["usage"]
        del metricbeat_doc["kibana_stats"]["usage"]
```

Reply:
Yea, this is better. I'll do that